sdcm/cluster.py

```diff
@@ -1863,6 +1864,7 @@ def install_mgmt(self, scylla_repo, scylla_mgmt_repo):
         """.format(mgmt_user=mgmt_user, rsa_id_dst=rsa_id_dst, rsa_id_dst_pub=rsa_id_dst_pub))  # generate ssh public key from private key.
         self.remoter.run('sudo bash -cxe "%s"' % ssh_config_script)
 
+        self.config_scylla_manager_yaml(segments_per_repair=10)
```

Comment:
What is the default currently?
1?

Reply:
The value does not exist in the manager yaml by default, but from what I found in scylla docs, it seems like the default is 1